test/e2e/webdriver/driver.js

```diff
@@ -1132,7 +1132,9 @@ class Driver {
         if (event.args.length !== 0) {
           const newError = this.#getErrorFromEvent(event);
 
-          const ignored = logBrowserError(ignoredConsoleErrors, newError);
+          const ignored = ignoredConsoleErrors.some((ignoredError) =>
+            newError.includes(ignoredError),
```

Comment:
we cannot look for the exact match, as sometimes the error has a changing id, like the `Event fragment with id transaction-added-07f2d240-499a-11ef-ab67-8da362fad529 does not exist.`error